site/src/routes/+page.svelte

```diff
@@ -3,12 +3,32 @@
 <script lang="ts">
     import Schedule from './Schedule.svelte';
     import CourseSearch from './CourseSearch.svelte';
+    import { onMount } from 'svelte';
 
     // Load data from `+page.ts`
     export let data;
 
     // Keep track of chosen sections
-    let selectedSections: ScheduleSelection[] = [];
+    let selectedSections: ScheduleSelection[];
+
+    // Retreive `selectedSections` from client local storage
+    onMount(() => {
+        if (typeof window !== 'undefined') {
+            const storedData = localStorage.getItem('selectedSections');
+            if (storedData) {
+                selectedSections = JSON.parse(storedData);
```

Comment:
I actually think this should check `data` to see if any of the information for a given course and section has changed. Otherwise, this could be retrieving outdated course and section information which would not be helpful. This should be fixed before this PR is merged.

Reply:
Done